xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/utilities/XWikiIcon.vue

```diff
@@ -19,47 +19,113 @@
  -->
 
 <template>
+  <img v-if="isImage" :src="url" :class="additionalClasses"/>
```

Comment:
Maybe it comes from the auto-formatting you are using, but it is considered as a [good practise](https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended) to keep each HTML attribute on a separate line, because this is easier to read and easier to keep track of changes in diffs. This would also ensure that the formatting stays consistent with what's already existing!

Reply:
I tend to keep the attributes on a single line as long as it stays below the column limit, but that's only a personal taste.
I'll update to stick to the best practice :)

Reply:
Great, thanks! :)